sidekiq_adhoc_job.gemspec

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'sidekiq_adhoc_job/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "sidekiq_adhoc_job"
+  spec.version       = SidekiqAdhocJob::VERSION
+  spec.authors       = ["Goh Khoon Hiang"]
+  spec.email         = ["gohkhoonhiang@gmail.com"]
+
+  spec.summary       = %q{Trigger jobs adhoc from Sidekiq web admin}
+  spec.description   = %q{Trigger jobs adhoc from Sidekiq web admin}
+  spec.homepage      = "https://github.com/gohkhoonhiang/sidekiq_adhoc_job"
+  spec.license       = "MIT"
+
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise "RubyGems 2.0 or newer is required to protect against " \
```

Comment:
eh how come now you put `\`? I thought you'd prefer single longer line more haha

Reply:
Because I copied-pasted this file 😛